util/src/main/scala/stryker4s/extensions/mutationtypes/MutationTypes.scala

```diff
@@ -7,7 +7,20 @@ import scala.meta.{Lit, Term, Tree}
 /**
   * Base trait for mutations. Mutations can be used to pattern match on (see MutantMatcher).
   */
-sealed trait Mutation[T <: Tree]
+sealed trait Mutation[T <: Tree] {
+  val mutationName: String
+}
+
+object Mutation{
+  // List of mutations
+  val mutations: List[String] = List[String](
+    "BinaryOperator",
```

Comment:
We could also change this static strings to this `classOf[BinaryOperator].getSimpleName` This would be a tiny bit safer when changing the mutator names but I agree the gain is little.

@hugo-vrijswijk @Wmaarts what do you think?

Reply:
I thought about it a bit and I agree. Thought about wrapping `classOf[T].getSimpleName` in a method or even in a trait, but that does feel like overthinking / overengineering things to me.

Reply:
i tried putting it into a method and that's impossible because of type erasure. Let's keep it like this